extension/services/deezer/player.js

```diff
@@ -124,6 +140,39 @@ this.player = (function() {
         iconVolumeParent.click();
       }
     }
+
+    async playSection({ query, thenPlay, section }) {
+      try {
+        let SECTION;
+        await this.search(query);
+        try {
+          if (thenPlay) {
+            const anchorNodes = await this.waitForSelector(".container h2 a", {
+              all: true,
+              timeout: 5000,
+            });
+            for (const anchorTag of anchorNodes) {
+              if (anchorTag.innerText === section) {
+                SECTION = anchorTag.parentElement.parentElement;
+              }
+            }
+            SECTION.querySelector("ul li button").click();
```

Comment:
This needs to check if `SECTION` is still undefined. Probably it should just raise an exception saying that the section couldn't be found.

Reply:
Ok